deploy/templates/azuredeploy.minimum.json

```diff
@@ -489,9 +489,9 @@
             ]
         },
         {
-            "comments": "Http Tunnel Consumer Group in IoT Hub",
+            "comments": "Test Consumer Group in IoT Hub",
```

Comment:
Is this `test` consumer group required ?

Reply:
No, but rather than creating more instability in the deployment I will handle this later when merging IAI and templates.